### PR TITLE
add JSP and controllers

### PR DESCRIPTION
**Pull request by olehedza**

### CODE REVIEW

File: src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            .getInstance(ShoppingCartService.class);
+    private final ShoppingCartDao cartDao = (ShoppingCartDao) INJECTOR
+            .getInstance(ShoppingCartDao.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long cartId = Long.parseLong(req.getParameter("cartId"));
+        ShoppingCart cart = cartDao.get(cartId).get();
+        List<Product> products = cartService.getAllProducts(cart);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java:31]: should we get all products or only products from our current shoppingCart?

File: src/main/java/mate/academy/webshop/controllers/DeleteProductFromCartController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    private ShoppingCartService cartService = (ShoppingCartService) INJECTOR
+            .getInstance(ShoppingCartService.class);
+    private ProductService productService = (ProductService) INJECTOR
+            .getInstance(ProductService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long productId = Long.parseLong(req.getParameter("productId"));
+        ShoppingCart cart = cartService.getByUserId(productId);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/webshop/controllers/DeleteProductFromCartController.java]: everything okey here?

File: src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java (hunk 1 above)
Comment (olehedza) [reply src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java:31]: only products from our current shoppingCart

File: src/main/java/mate/academy/webshop/controllers/DeleteProductFromCartController.java (hunk 2 above)
Comment (olehedza) [reply src/main/java/mate/academy/webshop/controllers/DeleteProductFromCartController.java]: Fixed.

File: src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    private final ShoppingCartService cartService = (ShoppingCartService) INJECTOR
+            .getInstance(ShoppingCartService.class);
+    private final ShoppingCartDao cartDao = (ShoppingCartDao) INJECTOR
+            .getInstance(ShoppingCartDao.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        Long cartId = Long.parseLong(req.getParameter("cartId"));
+        ShoppingCart cart = cartDao.get(cartId).get();
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java]: let's find user's cart by userId. Add shoppingCartService.getByUserId() method

Comment (dmytro-hryhoruk) [reply src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java]: you will always have user_id from session(next task) and you won't have to move cart_id from one controller to another

Comment (olehedza) [reply src/main/java/mate/academy/webshop/controllers/CheckoutOrderController.java]: Thank you. Got it, at last. Fixed.

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**